florent/config.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# VM
+export VM_NAME="vm-deb"
+
+# log
+export USER="florent"
+export USER_PWD=$'$6$6UHPrloRsJ2TotMt$vmK2e1v5FV/2xxd.patSn9dVN3NFJnLFy9d8ZdqMowqZkkjxnwxRXYVQSRlZZGPrXmcqLwqrzAX3wdDZncwCD.'
+export IP="192.168.122.66"
+export NC="nc -vz $IP 22"
```

Comment:
It's not a good practice to use commands in exported variables, you need to use a bash function for that and call that function instead of the variable.